### PR TITLE
GitHub Forks and Stars added in popular repository card

### LINKED ISSUE #54: [Feature]: Show stargazerCount and forkCount in popular repository card.
### Suggest an idea for this project ?

We have stargazerCount and forkCount available for the popular repository, we should show that in the repository card.

Something like this.

image

File reference

https://github.com/OpenSource-Journey/Your-Github-Contributions/blob/main/src/components/GithubPopularRepositories.tsx

### Please provide an example screenshot

_No response_

### Checklist

I have searched the existing issues to make sure this is not a duplicate, I have checked the documentation and found no answer

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mayukh551**
@Sachin-chaurasiya 

## Pull Request Checklist

Before submitting a pull request, please make sure that your changes meet the following criteria:

- [ ] Ensure that your code adheres to our coding conventions and is properly formatted.
- [ ] Test your changes extensively to ensure that they do not introduce any new bugs.
- [ ] Confirm that you agree to the code of conduct.

## Related Issues

If this pull request is related to any issues, please list them here.
Closes #54 

## Screenshots

If your changes include any visual elements, please provide screenshots or other visual aids to help reviewers understand your changes.
image

## Additional Notes

If there is anything else you would like to note about your changes or the review process, please add it here.

### CODE REVIEW

**Review by ShaileshParmar11 — APPROVED** [review]

File: src/components/GithubPopularRepositories.tsx [hunk 1]
```diff
@@ -55,6 +56,20 @@ const GithubPopularRepositories: FC<Props> = ({ repositories }) => {
...
+                    {repository.stargazerCount}
+                  </Text>
+                </Flex>
+                <Flex alignItems="center">
+                  <BiGitRepoForked />
+                  <Text color="GrayText" paddingLeft={1}>
+                    {repository.forkCount}
+                  </Text>
+                </Flex>
+              </Flex>
```
Comment (Sachin-chaurasiya) [inline src/components/GithubPopularRepositories.tsx:72]: This should be below the description.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: src/components/GithubPopularRepositories.tsx (hunk 1 above)
Comment (mayukh551) [reply src/components/GithubPopularRepositories.tsx:72]: ok

Comment (Sachin-chaurasiya) [reply src/components/GithubPopularRepositories.tsx:77]: @mayukh551 , Can you update the screenshots?

Comment (Sachin-chaurasiya) [reply src/components/GithubPopularRepositories.tsx:77]: @mayukh551, I meant in the PR description.

Comment (mayukh551) [reply src/components/GithubPopularRepositories.tsx:77]: Oh, ok

Comment (mayukh551) [reply src/components/GithubPopularRepositories.tsx:77]: @Sachin-chaurasiya updated the Screenshots

**Review by Sachin-chaurasiya — APPROVED** [review]
Great work @mayukh551  🙌🏻,
Thanks for the contributions ✨

**Comment by mayukh551** [comment]
Thnx for the opportunity 😄

**Pull request merged**